dask/kubernetes/Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM docker.io/rapidsai/rapidsai-core:21.06-cuda11.0-runtime-ubuntu18.04-py3.8
+FROM rapidsai/rapidsai-core:22.06-cuda11.5-runtime-ubuntu20.04-py3.9
 
-RUN source activate rapids \
-    && pip install gcsfs xgboost
+# Install required package for notebook and cluster control 
+RUN mamba install -n rapids -c conda-forge --freeze-installed -y kubernetes google-cloud-sdk gcsfs seaborn dask-kubernetes
+
+# Install kubectl autocompletion (optional)
+RUN apt-get update && apt-get install bash-completion && \
+    kubectl completion bash > /tmp/kube.completion.bash.inc && \
+    echo "source /etc/profile.d/bash_completion.sh" >> ~/.bashrc && \
+    echo "source /tmp/kube.completion.bash.inc" >> ~/.bashrc
```

Comment:
I probably wouldn't bother with this. Completion is nice to have but optional and it might confuse folks.